src/modules/home/vacancies/vacancies.styles.ts

```diff
@@ -45,6 +45,36 @@ export const SC = {
     }
   `,
 
+  LeftTopCloudImage: styled.img`
+    height: 100%;
```

Comment:
Create a css snippet with duplicated styles for cloud components.
It can be put in to shared/assets/styles/layout.ts file.
```
  const cloudStyles = css`
   height: 100%;
   position: absolute;
   top: 0;
   pointer-events: none;
   user-select: none;
  `
```
```suggestion
${cloudStyles}
left: 0;
transform: translate(-70%, -40%);
```